### DISCUSSION TITLE
How to simulate a supabase Provider login for my integration test?

### DISCUSSION THREAD

**Discussion by seenickcode** [supabase/supabase#3432] [Questions]
I am using the Flutter supabase package and am working on an integration test that should login with Google. It seems that even if I try to simulate Google authentication with a test Google user, this would be quite difficult and almost impossible because Google asks to verify with a secondary device (ie. "is this you trying to login?").

Are there any recommendations that folks may have for my use case?

Thanks.

**Comment by Kamsou** [comment]
Hello @seenickcode , I have the same problem. Were you able to find a solution ?

**Reply by seenickcode** [reply]
I'm sorry I do not recall. Perhaps it may be easier to run a full integration test that actually start a real emulator and hits a test Supabase account.
